src/test/kotlin/ratpack/example/kotlin/SiteSpec.kt

```diff
@@ -0,0 +1,72 @@
+package ratpack.example.kotlin
+
+import com.google.common.io.CharStreams
+import org.jetbrains.spek.api.Spek
+import ratpack.test.MainClassApplicationUnderTest
+import java.io.InputStreamReader
+import kotlin.test.assertEquals
+
+class SiteSpec : Spek({
+
+    val aut = MainClassApplicationUnderTest(Main::class.java)
```

Comment:
The only thing that I that is feasible for cleaning up the `aut` after the final test (without having to alter the code of  Spec), is to have a dummy fixture as the last entry.
It's a totally hacky solution, but it works.
Moreover, the original JUnit test did not contain a clean-up action